constants.gradle

```diff
@@ -19,7 +19,7 @@ project.ext {
     minSdkVersion = 14
     compileSdkVersion = 27
     targetSdkVersion = 27
-    buildToolsVersion = '26.0.2'
+    buildToolsVersion = '27.0.2'
```

Comment:
This is unrelated to the change. Can it please be removed?

Reply:
Sure.